### PR TITLE
Add acceptance tests for gzipped files

### PR DESCRIPTION
**Pull request by quartzmo**
Skip these failing tests for now.

[refs #1835]

### CODE REVIEW

File: google-cloud-storage/acceptance/storage/file_test.rb [hunk 1]
```diff
@@ -316,6 +319,34 @@
     uploaded.delete
   end
 
+  it "should download, verify, and decompress a gzipped file with default md5 verification" do
+    skip "Not working. See https://github.com/GoogleCloudPlatform/google-cloud-ruby/issues/1835"
+
+    lazy_bucket = storage.bucket bucket_public_test_name
+    lazy_file = lazy_bucket.file file_public_test_gzip_name
+
+    Tempfile.open ["google-cloud", ".png"] do |tmpfile|
```
Comment (jskeet) [inline google-cloud-storage/acceptance/storage/file_test.rb]: What's the .png for here? (Sorry, I don't know Ruby at all...)

File: google-cloud-storage/acceptance/storage/file_test.rb [hunk 2]
```diff
@@ -316,6 +319,34 @@
...
+      tmpfile.binmode
+      downloaded = lazy_file.download tmpfile
```
Comment (jskeet) [inline google-cloud-storage/acceptance/storage/file_test.rb]: Is md5 the default then? (Just trying to spot the differences between the tests.)

File: google-cloud-storage/acceptance/storage/file_test.rb (hunk 1 above)
Comment (quartzmo) [reply google-cloud-storage/acceptance/storage/file_test.rb]: Oops, that's a copy/paste from another test... it doesn't matter, but is not appropriate. Thanks.

File: google-cloud-storage/acceptance/storage/file_test.rb (hunk 2 above)
Comment (quartzmo) [reply google-cloud-storage/acceptance/storage/file_test.rb]: Yes, md5 is the default.

**Comment by quartzmo** [comment]
If not skipped, the current test output is:

```sh
  1) Error:
Google::Cloud::Storage::File::storage#test_0010_should download, verify, and decompress a gzipped file with default md5 verification:
Google::Cloud::Storage::FileVerificationError: The downloaded file failed MD5 verification.
    /Users/quartzmo/code/google/codez/gcloud-ruby/google-cloud-storage/lib/google/cloud/storage/file/verifier.rb:34:in `verify_md5!'
    /Users/quartzmo/code/google/codez/gcloud-ruby/google-cloud-storage/lib/google/cloud/storage/file.rb:1081:in `verify_file!'
    /Users/quartzmo/code/google/codez/gcloud-ruby/google-cloud-storage/lib/google/cloud/storage/file.rb:498:in `download'
    /Users/quartzmo/code/google/codez/gcloud-ruby/google-cloud-storage/acceptance/storage/file_test.rb:330:in `block (3 levels) in <top (required)>'
    /Users/quartzmo/.rbenv/versions/2.3.1/lib/ruby/2.3.0/tempfile.rb:295:in `open'
    /Users/quartzmo/code/google/codez/gcloud-ruby/google-cloud-storage/acceptance/storage/file_test.rb:328:in `block (2 levels) in <top (required)>'

  2) Error:
Google::Cloud::Storage::File::storage#test_0011_should download, verify, and decompress a gzipped file with crc32c verification:
Google::Cloud::Storage::FileVerificationError: The downloaded file failed CRC32c verification.
    /Users/quartzmo/code/google/codez/gcloud-ruby/google-cloud-storage/lib/google/cloud/storage/file/verifier.rb:42:in `verify_crc32c!'
    /Users/quartzmo/code/google/codez/gcloud-ruby/google-cloud-storage/lib/google/cloud/storage/file.rb:1082:in `verify_file!'
    /Users/quartzmo/code/google/codez/gcloud-ruby/google-cloud-storage/lib/google/cloud/storage/file.rb:498:in `download'
    /Users/quartzmo/code/google/codez/gcloud-ruby/google-cloud-storage/acceptance/storage/file_test.rb:344:in `block (3 levels) in <top (required)>'
    /Users/quartzmo/.rbenv/versions/2.3.1/lib/ruby/2.3.0/tempfile.rb:295:in `open'
    /Users/quartzmo/code/google/codez/gcloud-ruby/google-cloud-storage/acceptance/storage/file_test.rb:342:in `block (2 levels) in <top (required)>'
```

**Pull request merged**